### PR TITLE
adding signature lattes recipe

### PR DESCRIPTION
**Pull request by eva-kung**
Please edit the Main Menu and add in the new signature latte recipes

### CODE REVIEW

**Comment by eva-kung** [comment]
Please ensure the signature lattes recipe is listed on the Main Menu

File: recipes/signature_latte.txt
Comment (Hseju-s) [inline recipes/signature_latte.txt:1]: The "recipe" is spelled wrong in this.

**Review by Hseju-s — CHANGES_REQUESTED** [review]
The "recipe" is spelled wrong in this.

File: recipes/signature_latte.txt
Comment (eva-kung) [reply recipes/signature_latte.txt:1]: Ok modified

**Review by Hseju-s — APPROVED** [review]

**Pull request merged**